src/Model.js

```diff
@@ -9,9 +15,10 @@ export default class Model {
     }
 
     this.configure && this.configure()
-    this[exec_middlewares]('before', 'create')
-    // TODO: create model
-    this[exec_middlewares]('after', 'create')
+    this[execute_middleware]('before', 'create')
+    this[collection] = this.constructor.collection
+    this.id = this[collection].id()
```

Comment:
Hmm Mongo's stuff. Is this reusable if we manage to add support to SQL dbs? i.e. Is this Model logic or *Mongo's* Model logic?